.github/styles/Google/We.yml

```
@@ -2,10 +2,15 @@ extends: existence
 message: "Try to avoid using first-person plural like '%s'."
 link: 'https://developers.google.com/style/pronouns#personal-pronouns'
 level: warning
-ignorecase: true
+ignorecase: false
```

```suggestion
ignorecase: true
```

This is the Google developer styleguide rules, so we shouldn't change them unless we have good reason. What's the indication here for changing?

---

it keeps whining about US (United States abbreviation)